crates/utils/src/settings/structs.rs

```diff
@@ -56,6 +56,20 @@ pub struct Settings {
   pub opentelemetry_url: Option<String>,
 }
 
+#[derive(Debug, Deserialize, Serialize, Clone, SmartDefault, Document)]
+#[serde(default)]
+pub struct PictrsConfig {
+  /// Address where pictrs is available (for image hosting)
+  #[default("http://pictrs:8080")]
+  #[doku(example = "http://localhost:8080")]
+  pub url: String,
+
+  /// Set a custom pictrs API key. ( Required for deleting images )
+  #[default("API_KEY")]
+  #[doku(example = "API_KEY")]
+  pub api_key: String,
```

Comment:
I think you should put an actual (testing) api key here, to show how it looks. And not specify any default i guess.

Reply:
I'll remove the default, but I'm not sure what would be better than just `API_KEY` as a test.

Once this is merged, in lemmy-ansible, I plan to generate this key in the same way the postgres password is.